tests/config.c

```diff
@@ -230,6 +234,23 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
                cfg[cfg_idx].use_log = TRUE;
             }
          }
+         config_has_option("no-log")
```

Comment:
[Aside remark: In the spirit of what we discussed the other day to keep change-sets small, maybe ... in future, we should move-off these types of config options enhancements to a new issue. ??? Just seeking your thoughts, too , here .]

Reply:
Qs, though ... I couldn't figure this out ... `--no-log` seems to be an existing option?

is this code below a bug-fix, then, to set `use_log = FALSE` ?

Just asking for my own clarification.

Reply:
I thought it was missing, but happy enough to remove it. It's the default option anyway.

The way the boolean options work with our config, they don't take a true/false argument, they just exist or don't.